README.md

```diff
@@ -83,14 +83,47 @@ All the additive changes above must be accompanied by an explanation about how
 new and old senders and receivers that implement the version of the protocol
 before and after the change interoperate.
 
-No guarantees are provided whatsoever about the stability of the code that
-is generated from the .proto files by any particular code generator.
-
 ## Experiments
 
-In some cases we are trying to experiment with different features. In this case,
-we recommend using a "development" sub-directory instead of adding them to any
-protocol version. These protocols should not be used, except for
-development/testing purposes.
+### New Experimental Components  
+
+Sometimes we need to experiment with new components, for example to add a
+completely new signal to OpenTelemetry. In this case, to define new experimental
+components we recommend placing new proto files in a "development" sub-directory.
+Such isolated experimental components are excluded from
+above [stability requirements](#stability-definition).
+
+We recommend using
+`Development`, `Alpha`, `Beta`, `Release Candidate`
+[levels](https://github.com/open-telemetry/opentelemetry-specification/blob/main/oteps/0232-maturity-of-otel.md#maturity-levels)
+to communicate different grades of readiness of new components.
+
+Experimental components may be removed completely at the end of the experiment,
+provided that they are not referenced from any `Stable` component.
+
+Experiments which succeed, require a review to be marked `Stable`. Once marked
+`Stable` they become subject to the [stability requirements](#stability-definition).
 
-Another review must be conducted for in-development protocols to join the main project.
+### Experimental Additions to Stable Components
+
+New experimental fields or messages may be added in `Development` state to `Stable`
+components. The experimental fields and messages within `Stable components` are subject
+to the full [stability requirements](#stability-definition), and in addition, they must be
+clearly labeled as `Development` (or as any other non-`Stable` level) in the .proto file
+source code.
```

Comment:
If I have a new experimental field `experimentalField` and I decide to change its type from `Foo` to an different, incompatible type `Bar`, would I have to rename the field and use a new field number so that you abide by the stability requirements?

Reply:
Yes